tests/unit/sagemaker/tensorflow/test_estimator.py

```diff
@@ -21,8 +21,9 @@
 import pytest
 
 from sagemaker.estimator import _TrainingJob
-from sagemaker.tensorflow import defaults, model, TensorFlow
-from tests.unit import DATA_DIR
+from sagemaker.tensorflow import model, TensorFlow
+from sagemaker.tensorflow.defaults import TF_VERSION
+from tests.unit import DATA_DIR, PY_VERSION
```

Comment:
iirc `TF_VERSION` is `"1.10"` and `PY_VERSION` is `"py3"`, which isn't actually a valid combination if you tried it in an integ test. I'm guessing it doesn't matter in these tests, though.